### PR TITLE
Add tzlocal 1.1.2

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

**Comment by ericdill** [comment]
@regebro I am adding tzlocal to conda-forge for linux, mac and windows. This will allow users to easily install this with [conda] (http://conda.pydata.org/docs/) (conda install -c conda-forge tzlocal). The maintenance burden here is light as all testing and releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe which will be located at https://github.com/conda-forge/tzlocal-feedstock (and the other packages as <package>-feedstock) shortly after this PR is finished and merged. If you would like to be added as a maintainer (to make new releases for instance), I can happily add you to the maintainers list. If not, that is totally fine too!  Is there anyone else that you think would want to be pinged about maintaining this build?

**Comment by regebro** [comment]
No, I don't know anyone, sorry.

File: recipes/tzlocal/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+  run:
+    - python
+    - pytz
+
+test:
+  imports:
+    - tzlocal
+  requires:
+    - setuptools
```
Comment (jakirkham) [inline recipes/tzlocal/meta.yaml]: Could you please explain?

File: recipes/tzlocal/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  commands:
+    - cd {{ environ.SRC_DIR }} && python setup.py test
```
Comment (jakirkham) [inline recipes/tzlocal/meta.yaml]: Ah, I see, for the import.

Comment (jakirkham) [inline recipes/tzlocal/meta.yaml]: While this shouldn't cause any problems, this feels a little funky to me. Would it be alright if we move this to the script section instead?

Comment (ericdill) [inline recipes/tzlocal/meta.yaml]: I didn't think that `test` had a `script` section.  At least it doesn't appear to in the conda-build docs.  Should i just remove this part that executes the unit tests?

Comment (ocefpaf) [inline recipes/tzlocal/meta.yaml]: One liner here or a `run_test.py` is fine.

**Pull request merged**